src/client/pages/OfferNew/utils.ts

```diff
@@ -178,8 +179,14 @@ export const hasCurrentInsurer = (quote: OfferQuote): boolean =>
   Boolean(quote.currentInsurer)
 
 export const isStudent = (details: QuoteDetails) => {
-  const studentQuoteTypesSe = ['STUDENT_BRF', 'STUDENT_RENT']
-  if ('type' in details && studentQuoteTypesSe.includes(details.type)) {
+  const studentQuoteTypesSe = [
+    ApartmentType.StudentBrf,
+    ApartmentType.StudentRent,
+  ]
+  if (
+    'type' in details &&
+    studentQuoteTypesSe.includes(details.type as ApartmentType)
```

Comment:
Nope, the `type` property on the quotes is pretty bad since it can be `ApartmentType | NorwegianHomeContentsType | DanishHomeContentsType` and these types have no overlapping 😢 Hence we have to assert it unfortunately.

![image](https://user-images.githubusercontent.com/42962286/121322668-789a4280-c90f-11eb-9ae8-e1987fbd06ab.png)